src/views/login-service.jsx

```diff
@@ -0,0 +1,16 @@
+export async function login({ email, password }) {
+  const response = await fetch("https://reqres.in/api/login", {
```

Comment:
proposal: use built-in `fetch` instead of axios (one less dependency in the project)